### PR TITLE
094 make website dynamic

### LINKED ISSUE #94: Make Website Dynamic
## Description

Currently, the website is not mobile-friendly. Fix the CSS so that it's dynamic.

## Acceptance Criteria

- [ ] Website looks good on mobile.

## Definition of Done

- [ ] Acceptance criteria met
- [ ] PR approved by 2 others
- [ ] Merged into the main

### PR DESCRIPTION
**Pull request by h4yleysh4rpe**
## Context
Wanted to have the website to be responsive to various browser sizes, and modify all elements to avoid any weird behaviour. 

Closes #94 

## What Changed?
Changes on almost all pages (home/members/about/feedback). Per pages, changes are:
- Home - Responsive main header sizes, mission statement and image section changing to on top of each other, carousel arrows coming down under upcoming events header for small screen sizes. 
- Members - image and blurb stack for small screen sizes, chairs stack for small screen sizes, arrow blurbs removed, margins either side are even
- About - header blurb size changes, value cards stack on top of one another for small screen sizes and fit within edges of window
- FAQ and Feedback - active section changes from next to the FAQ/Contact buttons, to underneath. 

## How To Review
Please look at the above changes and ensure that all is running smoothly for various window sizes (can review commit wise if you want to ensure you have checked everything). Please also keep an eye out for any small other problems. Ignore the following as these are issues to be fixed later (after discussing with Nate although let me know if you want me to have a go with any of them soon) - 
- Header is not responsive to smallest screen sizes
- Member modal is not responsive to screen sizes
- Blobs on home page.

## Notes
Added a new breakpoint into tailwind.css (xs) to allow for a bit more flexibility on the smallest screen sizes.

### CODE REVIEW

File: strapi/src/index.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ module.exports = {
         lastname: "Team",
         username: "Younite",
         password: "Developer2023",
-        email: "ytao543@aucklanduni.ac.nz",
+        email: "543@aucklanduni.ac.nz",
```
Comment (Oculux314) [inline strapi/src/index.js]: Yo! Was this an accidental deletion?

**Review by Oculux314 — COMMENTED** [review]
Looks awesome and I'm almost ready to approve - just letting you know that on medium-sized laptop screens the text gets a bit big. Let me know if you want to fix or not fix in this PR and then I'm happy to approve :)

E.g. home (I'm on a 14")
image

**Review by Oculux314 — APPROVED** [review]
Looks good per our discussion yesterday :D

Be aware of this bug I'm documenting here: 
image
But I'm pretty sure this is caused by the header which we'll turn into a Hamburger menu soon anyway so we can leave it :)

File: strapi/src/index.js (hunk 1 above)
Comment (h4yleysh4rpe) [reply strapi/src/index.js]: Fixed

**Comment by Oculux314** [comment]
@zozzzC @jessicazeng13 quick bump on this, Hayley's so excited to merge this in she's fallen sick :)

**Comment by zozzzC** [comment]
Hiya, just checking if this was something that was needed to be fixed or not, but in the small screen sizes the About page cards are quite big--is there a reason for that or is it alright that way? 
image

**Review by jessicazeng13 — APPROVED** [review]
Everything looks great and is working on my end! I think there are minor styling changes that can be on the values page when it is stacked on top of each other, but I don't think that this is the job for this ticket so don't worry! Great work this looks sick!!

**Pull request merged**